taf/api/metadata.py

```diff
@@ -4,12 +4,12 @@
 from logdecorator import log_on_end, log_on_error
 from taf.api.utils._git import check_if_clean, commit_and_push
 from taf.exceptions import TAFError
-from taf.git import GitRepository
 from taf.keys import load_signing_keys
 from taf.constants import DEFAULT_RSA_SIGNATURE_SCHEME
 from taf.messages import git_commit_message
 from taf.repository_tool import Repository, is_delegated_role
 from taf.log import taf_logger
+from taf.taf.auth_repo import AuthenticationRepository
```

Comment:
Getting this error here (should be `taf.auth_repo`) 
`from taf.taf.auth_repo import AuthenticationRepository
ModuleNotFoundError: No module named 'taf.taf'`